### PR TITLE
Changed file1

### PR DESCRIPTION
**Pull request by prathapreddy123**
Changed file 1

### CODE REVIEW

File: file1.txt [hunk 1]
```diff
@@ -1 +1,2 @@
-"This is file1" 
+"This is file1"
```
Comment (pradeepzreddy) [inline file1.txt:1]: Great work Prathap

Comment (prathapreddy123) [reply file1.txt:1]: Thanks

File: file1.txt [hunk 2]
```diff
@@ -1 +1,2 @@
...
+"prathap added second line"
```
Comment (pradeepzreddy) [inline file1.txt:2]: add more text

Comment (pradeepzreddy) [inline file1.txt:2]: just one more comment

**Review by pradeepzreddy — APPROVED** [review]
approving

**Pull request merged**